Src/Sarif.PatternMatcher.Cli/Program.cs

```diff
@@ -20,6 +22,15 @@ internal static int Main(string[] args)
         {
             Console.OutputEncoding = Encoding.UTF8;
 
+            args = EntryPointUtilities.GenerateArguments(args, FileSystem, new EnvironmentVariables());
```

Comment:
>GenerateArguments [](http://example.com/codeflow?start=39&length=17)

question for the logic,
if there is a ResponseFile arg, we expand and return all.
But if not, do we miss out all the normal args user input?
the args is replaced after this line execution and will be used below line "Parser.Default.ParseArguments"

Reply:
I see GenerateArguments do keep the normal args and adds the ones from ResponseFile. Not an issue.

Reply:
Yes! It's actually very flexible, you can arbitrarily insert one or more response files onto the same command-line:

myapp.exe arg1 @responsefile1.rsp anotherArg @responseFile2.rsp